test/integration/infrastructure/infrastructure_test.go

```diff
@@ -408,7 +408,7 @@ func generateNamespaceName() (string, error) {
 		return "", err
 	}
 
-	return "shoot--it--" + suffix, nil
+	return "shoot--it-aws--" + suffix, nil
```

Comment:
Can we change this to something completely different, please? This is highly confusing and conflicts with potential project names.
What about `aws-infrastructure-it--` or something similar?

Reply:
Yeah, right. Done :)